src/Tiqr/Legacy/TiqrService.php

```diff
@@ -130,6 +122,7 @@ public function generateEnrollmentKey(string $sari): string
      */
     private function storeEnrollmentKey(string $key): void
     {
+        $this->initSession();
         $keys = [];
         if ($this->session->has(self::ENROLL_KEYS_SESSION_NAME)) {
```

Comment:
Maybe use $this->requestStack->getSession()->has ???

Reply:
For now we work with an init construction. I don't feel comfortable changing this here.